src/java.desktop/share/classes/javax/swing/table/JTableHeader.java

```diff
@@ -935,7 +935,7 @@ protected class AccessibleJTableHeaderEntry extends AccessibleContext
             private JTable table;
 
             /**
-             *  Constructs an AccessiblJTableHeaaderEntry
+             *  Constructs an AccessibleJTableHeaderEntry
```

Comment:
There's an extra space in front of "Constructs"..please remove that too..

Reply:
Updated.